### DISCUSSION TITLE
700 series controllers becoming unresponsive / jammed

### DISCUSSION THREAD

**Discussion by rohrsh** [zwave-js/zwave-js#7069] [General]
Hello everybody. I'm struggling to get my network working reliably and would greatly appreciate any suggestions.  

I started with a small Zwave-JS-UI network to test a few devices, and that was operating just fine for the last few months. Last week I then migrated the rest of my ~50 devices over from Fibaro Home Center 2. 

**Messages are slow to execute (if at all) and Home Assistant regularly reports the controller is _Jammed_**

- Diagnostics on many nodes show that my controller can ping a node, but all return pings fail (See (b). Similar to https://github.com/zwave-js/node-zwave-js/discussions/5179
- Logs show some devices repeatedly updating the same value to the controller (See (a), which led me to this https://github.com/zwave-js/node-zwave-js/issues/6284

Situation:
- Zwave-JS-UI 9.14.3 docker on Raspberry Pi 3b. It was originally in a NUC docker, but I moved to the Pi temporarily so I could roam around to do exclude/includes
- Home Assistant websocket - but I'm running all my tests in Zwave-JS-UI
- Silicon Labs 700 stick. Firmware was originally 7.19.2 but I've since upgraded to 7.22.3 without any improvement. Could I just be suffering from the 700 series firmware bug? 
- 2m USB extension cable 
- I have rediscovered neighbours and re-interviewed many times
- Most of the devices are within 20m of the controller. The mesh seems very dense - some nodes have ~20 neighbours. Electrician often placed 6x pucks jammed into one plastic box. I see lots of red 9.6kb links (See (c)
- Devices are mostly Fibaro Dimmers and Relays, which are ~5 years old, I think 300 series protocol
- Value update reports set to reasonable thresholds: e.g. my dimmers only report data on 10% power change, or hourly
- I have read the troubleshooting docs here https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index
- My devices did work last week on the Fibaro Home Center 2 controller

**Logs attached (d). Towards the end of the log you'll see me fiddling with Node 20 (ok) and then it jams when adjusting Node 31.** 

**Thank you for any help or suggestions.** 

(a)
<img width="1075" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/38928979/ede1583c-11b5-46b3-9b30-7b6fada5f05b">

(b) 
<img width="765" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/38928979/196248a3-798a-4551-bd25-9c58c329138b">

(c)
<img width="688" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/38928979/370d89df-3b2d-43a6-8dc9-8db4c215c550">

(d)
[zwavejs_2024-06-27.log](https://github.com/user-attachments/files/15997642/zwavejs_2024-06-27.log)

**Comment by rohrsh** [comment]
Some of the routing choices seem pretty odd: e.g. this node 45 is in a rather central location and has dozens of neighbours to choose from, I'm sure there's a better path than what has been chosen. 

I wonder if the controller is restarting while testing routes, which delivers bad data, which leads to odd routing decisions. 

<img width="787" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/38928979/3bad4487-9013-4454-9997-2d80de24eaa0">

**Comment by rohrsh** [comment]
<img width="308" alt="image" src="https://github.com/zwave-js/zwave-js-ui/assets/38928979/68f08cb1-a575-4709-9630-0b3515b13347">

**Comment by rohrsh** [comment]
Options I'm considering
- Setting priority/return routes
- Get a 500 series controller
- Split into two Z-wave networks

**Comment by Idaho947** [comment]
It a known bug on the firmware. Just have to wait. But silicon lab is on it for one year....

**Comment by AlCalzone** [comment]
See https://github.com/zwave-js/node-zwave-js/discussions/6874, https://github.com/zwave-js/node-zwave-js/discussions/6512, https://github.com/zwave-js/node-zwave-js/discussions/6911 for the jammed/unresponsive issue.

FWIW, you're on firmware 7.21.3. From what I've seen, 7.21 is in a pretty bad state right now, even worse than 7.19.
7.22.x is limited to 800 series controllers, but it is supposed to include a few fixes for the lockup issues.

I'll take a look at the logs tomorrow to see if anything else sticks out, since you seem to have followed the typical troubleshooting steps already.
My personal experience is to avoid 700 series controllers though. Either 500 or 800 series IMO, 800 has much better range, but you're at the mercy of Silabs to fix the issues.

**Comment by webmogul1** [comment]
I am using the ZOOZ 700 stick with firmware v7.18.3 for quite some time without apparent issues.

**Comment by webmogul1** [comment]
> I wonder if the controller is restarting while testing routes, which delivers bad data, which leads to odd routing decisions.
> 
Without any expertise on my part, I would say that's the problem. At least, the same thing happened to me when I switched to an 800 stick and firmware 7.21.3

**Comment by AlCalzone** [comment]
Given how long we've seen complaints about bad routes (and the fact that I've seen the code responsible for choosing routes), I would say the two are not related. The routing algorithm is just terrible tbh.

**Comment by rohrsh** [comment]
>  The routing algorithm is just terrible tbh.

Any best practice guides for messing with priority/return routes? I saw this [DrZwave guide. ](https://drzwave.blog/2021/07/13/z-wave-mesh-priority-routes-explained/) which seems to suggest only setting a priority route for nodes that you are certain can reach your controller.

**Comment by rohrsh** [comment]
> My personal experience is to avoid 700 series controllers though. Either 500 or 800 series IMO, 800 has much better range, but you're at the mercy of Silabs to fix the issues.

Slim pickings here in Australia! 

The 800 series Zooz ZST39 doesn't have an AU variant for sale. Could I buy the US model and simply set to AU region in ZwaveJS? Or will I need special firmware? AU-specific SAW filter? I don't need the LR functionality. [Some Redditors](https://www.reddit.com/r/homeassistant/comments/163hamh/zooz_800_series_zwave_change_region/) here mentioned under-the-table AU firmware via thesmartesthouse.com

I dd try ordering a local Aeotec Gen5+, but the vendor helpfully delivered an 'upgraded' Z-Stick 700 instead... 

Is the 500 series [Z-Wave RaZberry Card V2](https://smarthomedirect.com.au/product/zme-z-wave-razberry-card-v2-gateway/?v=322b26af01d5) usable? I can see AU stock of that

**Comment by zackbcom** [comment]
You could check out this Silicon Labs [Z-Wave 800 Series Development Kit](https://www.silabs.com/development-tools/wireless/z-wave/z-wave-800-dev-kit?tab=overview). I have two devices. One flashed to be a controller and one flashed to be a zniffer. You can flash different country firmwares. Look into that. Also you can upgrade to any new firmware without having to wait for a vendor to roll their own out.

**Comment by AlCalzone** [comment]
> Could I buy the US model and simply set to AU region in ZwaveJS? Or will I need special firmware? AU-specific SAW filter?

AFAIK (but I might be wrong), the hardware is the same for all regions, they just come with the firmware pre-set to the correct region. At least I am regularly switching my US ZST39 to EU frequencies.

> Is the 500 series [Z-Wave RaZberry Card V2](https://smarthomedirect.com.au/product/zme-z-wave-razberry-card-v2-gateway/?v=322b26af01d5) usable? I can see AU stock of that

It works, but not sure if the firmware supports SmartStart etc. Upgrades require using Z-Wave.me software.

**Comment by Botched1** [comment]
> * Silicon Labs 700 stick

I also use a UZB-7 from SiLabs on my ~100 device network. 7.18.3 is the only firmware that I've tried that works pretty much 100% correctly. Occasional jammed status report, but it is VERY rare for me (<1/week).

**Comment by rohrsh** [comment]
Ordered a new Zooz 700 and it arrived with 7.19.2 firmware
Tried to downgrade my Silabs 700 via Windows Z-wave PC Controller, but the bootloader said no (Error 0x18)

So I've started a second network on my Zooz stick. Will migrate a few devices over each day and see if there happens to be some magic number of devices, or a specific device, that triggers the Jamming.

**Comment by rohrsh** [comment]
So I've been migrating my network over to a new 7.19.2 Zooz 700 stick, a dozen devices each day. Got to 24 nodes and looked ok - I was testing with the ping diagnostics and individual node control. 

However, things turned bad when I tried to control multiple (3 -  _Edit: 4_) lights at once (via Home Assistant.) Some lights would take 10+ seconds to implement the command. 

Maybe I'm hitting this? https://github.com/zwave-js/node-zwave-js/issues/5471 ... although I run with no security. 

Logs appear to show way too much traffic: here's a sample of me turning off and then on 3 lights simultaneously. They are Fibaro dimmers which report power changes. 
[zwavejs_2024-07-13--turnonoff.log](https://github.com/user-attachments/files/16200797/zwavejs_2024-07-13--turnonoff.log)

Here's my full day of logs for completeness also. The driver appears to crash at one point in my testing. 
[zwavejs_2024-07-13.log](https://github.com/user-attachments/files/16200798/zwavejs_2024-07-13.log)

Is this still the same 700 series firmware bug and I just have to wait?

**Comment by rohrsh** [comment]
Incidentally, two of the lights in this group of 4 are not currently directly connected to the controller. Could exacerbate the problem. 

e.g. Living Alcove, Living Dining Downlights [top right of map]

<img width="853" alt="image" src="https://github.com/user-attachments/assets/2848e444-f538-4cda-a567-c5f9754a37a4">

**Comment by zwave-js-assistant** [comment]
This issue has not seen any recent activity and was marked as "stale 💤".
Closing for housekeeping purposes... 🧹

Feel free to reopen if the issue persists.

**Comment by rohrsh** [comment]
Please don't close until we see new firmware to fix the 700 series. The fixes out this week are only for 800 series for now.

**Reply by AlCalzone** [reply]
I've moved the issue to discussions. This is nothing we can do anything about, so it should not be under issues.

**Reply by rohrsh** [reply]
Comment from Silicon Labs employee: 

> The 7.22 SDK contains improvements for the controller lockup issue, but as others pointed out, this applies to the 800 series devices only. The 700 series devices will be supported in the upcoming 7.21.x releases. The next patch release (7.21.4) could be expected in the **middle of August** and will contain improvements for the lockup.

https://community.silabs.com/s/question/0D7Vm000003MTo1KAG/detail?language=en_US

**Comment by hughc-hub** [comment]
hi... I've seen that v7.21.4 is available for 700 series.. 
https://aeotec.freshdesk.com/support/solutions/articles/6000263744-update-z-stick-7-with-z-wavejs-ui-v7-18-8-v7-21-4-

Did someone test it? would you recommended vs v7.18.8 ?
I've just updated mine from 7.17.2 to 7.18.8 and I feel the network is a little bit better now.. 

thanks a lot!

**Comment by MusinoPetr** [comment]
Adding another large-network data point for the 700/800-series controller lock-up.

Controller: Aeotec Z-Stick 10 Pro (SiLabs 800-series), firmware V3.0 (SDK ~7.23–7.24, latest available).
Setup: Home Assistant Core 2026.8.1, Z-Wave JS add-on 1.7.0 / driver 15.27.0, EU 868 MHz.
Network: ~184 nodes, mostly Fibaro (FGS-223, FGBS-222, FGR-22x, FGWP-102) + Qubino ZMNHXD + STEINEL IS 140-2. Ran 7 years flawlessly on a Fibaro 500-series HC; problems started only after switching to the Z-Stick 10 Pro.
Symptoms: controller enters jammed several times/day (1 s to ~29 min). Zniffer shows two signatures: (1) controller-side lock-up — air quiet, node 1 transmits 0 frames, recovers only on add-on restart; (2) RF collision storm, CRC 10–15 %. Nodes (esp. STEINEL) go dead/mute and only a physical power-cycle revives them.

Measured: one 29-min jam = 5 485 frames/30 min, controller transmitted only 5 frames the whole time. Peak storm hour ~24 000 frames/h at ~15 % CRC. Already on latest firmware; throttling chatty nodes only reduces frequency.

Is there any newer firmware or driver mitigation for the 800-series lock-up on large busy EU networks, or is splitting to a second/500-series controller the only reliable path? Happy to provide logs / NVM / captures.

**Reply by AlCalzone** [reply]
The most recent Z-Wave SDK (8.0.x) includes several fixes that might help with these issues. I don't think Aeotec have a build available with that though.

Out of curiosity, how busy is your network when not jammed? Any routes on lower speed (40 or even 9.6 kbps)?

I _think_ part of the larger problem is that older devices, including controllers, didn't really follow the regulations when it comes to duty cycle (clear channel assessment) and just transmitted anyways. 700/800 series are more strict, which manifests as short-time jams (not sure what was with that 29 minute situation though).
The earlier firmwares from those series had a bunch of bugs that also manifested the same way. On the more recent firmwares I'd say most jams are actual indications of a too busy RF environment.

**Reply by MusinoPetr** [reply]
Thanks — I went through the Zniffer properly. Figures from a clean 24 h (yesterday, no anomalies):

Busyness: ~154 frames/min on average over the day; ~79/min during the deep-night idle. Nothing extreme in terms of frame count.

But the data rates are the real story — from the per-frame data-rate decode:

100 kbit/s: 18 %
40 kbit/s: 63 %
9.6 kbit/s: 15 %
So ~78 % of all traffic runs at 40 k or 9.6 k — almost nothing communicates at the full 100 k, and the 9.6 k share is largely explorer / route-resolution frames (a lot of nodes clearly struggle to find a route). Channel occupancy is therefore dominated by slow traffic — which lines up exactly with your CCA point: it's the low-speed airtime keeping the channel busy and making the 800-series back off, not the raw frame count.

Top talkers/day: a Qubino ZMNHXD meter (~8 k frames), two Fibaro FGBS-222 Smart Implants (~4.4 k / 2.5 k, one of them spamming "no handler / unknown notification 0xca"), a solar-temperature FGBS, and other meters (which we've since throttled).

The network is ~184 mostly-Fibaro nodes, EU 868, Z-Stick 10 Pro on the latest firmware. Given that SDK 8.0.x isn't available from Aeotec yet: does this profile (lots of low-speed routes + heavy explorer traffic) point to a routing problem worth forcing (heal / priority routes to push 100 k where possible), or do these older Fibaro devices simply not negotiate 100 k, so the mesh is inherently slow? Happy to share the capture as well.

**Reply by AlCalzone** [reply]
> ~154 frames/min on average over the day
> ~79/min during the deep-night idle

I would say that is far too much. Aim for seconds/frame, not frames/second. Although the numbers are probably inflated by retries etc.

> So ~78 % of all traffic runs at 40 k or 9.6 k 

Not good.

> and the 9.6 k share is largely explorer / route-resolution frames (a lot of nodes clearly struggle to find a route)

Probably a self-amplifying cycle. Nodes not being able to communicate because of noise generate more noise through retries and explorer frames. 9.6k communication takes extremely long, blocking airtime for others.

> do these older Fibaro devices simply not negotiate 100 k

AFAIK they should support it, unless they are really old (300 series and older).

---

Plan of action IMO:
1. Check the background RSSI of your controller. It should be between -100 and -110 dBm ideally. Is it on a USB extension cable, away from other USB ports and metal surfaces?
2. quiet down the heavy talkers first. I'm definitely not a fan of energy metering on Z-Wave Classic - for obvious reasons.
3. once you've done that, identify close by nodes that need help staying on 100 kbps. If there are some that should be close and new enough to support 100 kbps reliably, they might need a priority route. Work outwards from the controller.

_Side note: I'm not sure how good the range on the Z-Stick 10 Pro is. I know the Z-Stick 7 had terrible range. Might be worth looking at the https://www.home-assistant.io/connect/zwa-2/ to eliminate repeaters as much as possible._

**Reply by MusinoPetr** [reply]
Thanks — this was a genuinely useful plan, and it's already paid off partially.

What we've done so far:

Background RSSI check: channel 0 (100k) sits around −96 dBm, the other channels are down at the −128 noise floor. So no constant external interference — the noise really is self-generated, exactly the self-amplifying cycle you described (retries + explorer frames on a busy channel). The controller's already on a 3 m USB extension, away from the machine.
Quieted the heavy talkers — throttled the energy meters right down. (And yes… energy metering on Z-Wave Classic. Agreed.)
Separately, we tracked down why a batch of STEINEL PIRs kept going mute/dropping after every restart: a polling / stale-value refresh sweep was jamming their TX stack, plus a leftover "gateway check" behaviour from the old Fibaro controller had them flooding. We removed both — and they've now survived 4 restarts without dropping once. A nice partial win.

And thanks a lot for the ZWA-2 tip — we've ordered one. Getting more nodes to talk directly to the controller via the external antenna (and working outward from there to cut the low-speed routing) is exactly the angle we want to try next.

Really appreciate you taking the time on this.

**Reply by AlCalzone** [reply]
> −128 noise floor.

Thats not the noise floor - that is the firmware unable to measure the background noise. I'm told this happens when the radio is busy while measuring.
